shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -751,11 +870,12 @@ - (void)pressesBegan:(NSSet<UIPress*>*)presses
            withEvent:(UIPressesEvent*)event API_AVAILABLE(ios(9.0)) {
   if (@available(iOS 13.4, *)) {
     for (UIPress* press in presses) {
-      [_textInputDelegate handlePressEvent:[[FlutterUIPressProxy alloc] initWithPress:press
-                                                                            withEvent:event]
-                                nextAction:^() {
-                                  [super pressesBegan:[NSSet setWithObject:press] withEvent:event];
-                                }];
+      [_textInputDelegate
+          flutterTextInputView:self
+              handlePressEvent:[[FlutterUIPressProxy alloc] initWithPress:press withEvent:event]
```

Comment:
This is a leak, FlutterUIPressProxy should be released here or autoreleased.

Reply:
Ah...this is actually something new that was introduced when I merged in master last time...I updated it to pass in the flutterTextInputView, but I'll go ahead and add these to the autorelease pool as well.